app/assets/stylesheets/dm/pages/_diffusion_map.scss

```diff
@@ -17,3 +17,15 @@
     visibility: hidden;
   }
 }
+
+.map-filters-accordion {
+  h4 {
+    @include u-maxw('full', !important);
+  }
+
+  .usa-accordion__content {
+    border-bottom: 0.25rem solid #f0f0f0 !important;
```

Comment:
i believe `#f0f0f0` is `$theme-color-base-lightest`

Reply:
Okay, I changed it 😄